tools/tck-api/src/main/scala/org/opencypher/tools/tck/api/Scenario.scala

```diff
@@ -203,6 +213,10 @@ case class Scenario(categories: List[String], featureName: String, number: Optio
           throw ScenarioFailedException(msg, error)
       }
     }
+
+    def withParameters(additionalParameters: Map[String, CypherValue]): ScenarioExecutionContext = {
```

Comment:
I wonder if this name captures better what the function is doing:

```suggestion
    def addParameters(additionalParameters: Map[String, CypherValue]): ScenarioExecutionContext = {
```